Api/Controllers/RestAPI/UserController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Application.Interfaces;
+using Domain.Entities.Attribute;
+using Domain.Entities.User;
+using Microsoft.AspNetCore.Mvc;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Api.Controllers.RestAPI
+{
+    [Route("api")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly IUserServices _userServices;
+        private readonly IGenericRepository<Client> _clientRepo;
+        private readonly IGenericRepository<UserEntity> _userRepo;
+        public UserController(IUserServices userServices, IGenericRepository<Client> clientRepo, IGenericRepository<UserEntity> userRepo)
+        {
+            _userServices = userServices;
+            _clientRepo = clientRepo;
+            _userRepo = userRepo;
+        }
+        // GET: api/<UserController>
+        [Route("Client")]
+        [HttpGet]
+        public IEnumerable<string> Get()
```

Comment:
asp.net have problem when 2 controller have methods that the same name, Give methods in this class a more specific name